client/app/ui-components/clipboard/clipboard.component.js

```diff
@@ -12,6 +12,11 @@
             };
         }
 
+        $onInit () {
+            if (!this.rows) {
+                this.rows = 1;
```

Comment:
`numberOfRows` or `rowCount` or whatever, but `rows` should contains rows and not a number

Reply:
I hesitated between rowCount and rows. I decided to go with rows to reflect the html attribute of the textarea: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea

Because it is exactly a 1-to-1 matching between my component and the html of the textarea. When naming variable, I don't like to have 2 different names for the exact same thing.